### PR TITLE
chore: remove LouisXhaferi from codeowners

### PR DESCRIPTION
**Pull request by LouisXhaferi**
**Description**
As I've practically not put any all that much time into modelina, since I first joined as a contributor, and won't be able to do so in the future, I'd step down as contributor/codeowner etc. 

I'll probably do the same for the TSC, as my focus shifted away from cloud and messaging related stuff.

### CODE REVIEW

File: README.md
Comment (LouisXhaferi) [inline README.md:1]: pain, random reformatting by IntelliJ all over the place.

File: .all-contributorsrc [hunk 1]
```diff
@@ -464,19 +464,6 @@
...
-      "login": "LouisXhaferi",
-      "name": "Louis Xhaferi",
-      "avatar_url": "https://avatars.githubusercontent.com/u/52397677?v=4",
-      "profile": "https://github.com/LouisXhaferi",
-      "contributions": [
-        "code",
-        "test",
-        "doc",
-        "example",
-        "maintenance"
```
Comment (jonaslagoni) [inline .all-contributorsrc:477]: Just remove maintenance 🙂 You still contributed with the rest 🙂

**Review by jonaslagoni — CHANGES_REQUESTED** [review]
Sad to see you go, but best of luck in your new adventure! You are free to come back any time 👍 

Only change the code owners file, no need to remove all traces of you contributing to the repo 😄

**Comment by LouisXhaferi** [comment]
> Sad to see you go, but best of luck in your new adventure! You are free to come back any time 👍
> 
> Only change the code owners file, no need to remove all traces of you contributing to the repo 😄

Thank you!

**Review by jonaslagoni — APPROVED** [review]

**Pull request merged**